validphys2/src/validphys/pdfplots.py

```diff
@@ -188,19 +189,28 @@ def _warn_any_pdf_not_montecarlo(pdfs):
 
 class ReplicaPDFPlotter(PDFPlotter):
     def draw(self, pdf, grid, flstate):
+        labels = flstate.labels
+        handles = flstate.handles
         ax = flstate.ax
         next_prop = next(ax._get_lines.prop_cycler)
         color = next_prop['color']
-        flavour_grid = grid.select_flavour(flstate.flindex)
-        stats = flavour_grid.grid_values
+        stats = grid.select_flavour(flstate.flindex).grid_values
         gv = stats.data
         ax.plot(grid.xgrid, gv.T, alpha=0.2, linewidth=0.5,
                 color=color, zorder=1)
-        ax.plot(grid.xgrid, stats.central_value(), color=color,
-                linewidth=2,
-                label=pdf.label)
+        cv_line = ax.plot(grid.xgrid[0:1], stats.central_value()[0:1], 
+                color=color, linewidth=2)
+        handle = cv_line[0]
+        labels.append(pdf.label)
+        handles.append(handle)
```

Comment:
@scarlehoff @RoyStegeman This PR is adding a plot that is used in very limited circumstances, but that makes implementing new subclasses measurably more difficult as demostrated in this bit. I am not too happy with that as per the existing discussion. I see no pressure to merge it.

Reply:
> that makes implementing new subclasses measurably more difficult as demostrated in this bit

How does this bit demonstrate that?

Reply:
(also, the very limited circumstances are already in the arxiv, which is more than one can say of many of the other plots that we have)

Reply:
The only other option that I see is to copy the whole of the `draw` method of `ReplicaPDFPlotter`  into `MixMatchPDFPlotter` instead of reusing `ReplicaPDFPlotter`.